uri-sexp.opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "anil@recoil.org"
+authors: ["Anil Madhavapeddy" "David Sheets" "Rudi Grinberg"]
+license: "ISC"
+tags: ["url" "uri" "org:mirage" "org:xapi-project"]
+homepage: "https://github.com/mirage/ocaml-uri"
+bug-reports: "https://github.com/mirage/ocaml-uri/issues"
+dev-repo: "git+https://github.com/mirage/ocaml-uri.git"
+doc: "https://mirage.github.io/ocaml-uri/"
+synopsis: "An RFC3986 URI/URL parsing library"
+description: """
+ocaml-uri with sexp support
+"""
+depends: [
+  "uri"
+  "dune" {build & >= "1.2.0"}
+  "ppx_sexp_conv" {build & >= "v0.9.0"}
```

Comment:
i _think_ ppx nowadays sometimes comes with runtime library, and we should drop the `build` attribute here